### ISSUE TITLE
Warning that react is not installed when I do in fact have 0.14.3 installed.

### ISSUE THREAD

**Issue by jpierson** [GriddleGriddle/Griddle#298]
When running `npm install griddle-react` I see the following error.

> griddle-react@0.2.16 requires a peer of react@^0.13.3 but none was installed.

I do however have ReactJS 0.14.3 installed.

```
> npm list react
gmaw@2.31.4 C:\Users\jeff\source\global-market-account\gmaweb
├── UNMET PEER DEPENDENCY react@0.14.3
└─┬ react-intl@1.2.2
  └── react@0.13.3

npm ERR! peer dep missing: react@^0.13.3, required by griddle-react@0.2.16
```

**Comment by jpierson** [comment]
It looks like running `npm install griddle-react@0.3.0 --save` with the 0.3.0 version set explicitly got around the issues. Does 0.2.16 not work with newer version of react past 0.14?

**Comment by Neta-Alon** [comment]
I got the exact same issue. Very weird.

**Comment by tdamsma** [comment]
^0.13.1 is compatible with (^) 0.13.X, not with 0.14.X, because the minor releases before 1.0.0 afe often not compatible (even though semver states they should be). After 1.0.0 ^ looks allows minor releses, so 1.0.1 is compatible with 1.1.0

**Comment by jpierson** [comment]
@tdamsma so it sounds like this is intentional. Out of curiosity is this default npm semver behavior or something specific with the griddle-react package?

**Comment by ryanlanciaux** [comment]
We are not currently using actual semver but will be going that route once 1.0.0 is released.

**Comment by tdamsma** [comment]
This is npm behaviour: 

> Caret Ranges ^1.2.3 ^0.2.5 ^0.0.4

> Allows changes that do not modify the left-most non-zero digit in the [major, minor, patch] tuple. In other words, this allows patch and minor updates for versions 1.0.0 and above, patch updates for versions 0.X >=0.1.0, and no updates for versions 0.0.X.

> Many authors treat a 0.x version as if the x were the major "breaking-change" indicator.

https://docs.npmjs.com/misc/semver#caret-ranges-1-2-3-0-2-5-0-0-4

**Comment by ryanlanciaux** [comment]
This should be fixed with Griddle 1.0.
